examples/deployment/scripts/transfer/gold_sc.js

```diff
@@ -5,7 +5,7 @@ async function run(runtimeEnv, deployer) {
   const johnAccount = deployer.accountsByName.get("john-account");
   const bobAccount = deployer.accountsByName.get("bob-account");
 
```

Comment:
Same here, let's add a short comment at the top of the file.

Reply:
Done